docs/reference/query-languages/esql/_snippets/functions/examples/match.md

```diff
@@ -15,6 +15,10 @@ FROM books
 | 2883 | William Faulkner |
 | 3293 | Danny Faulkner |
 
+```{applies_to}
+stack: ga 9.1.0
+```
```

Comment:
Does it make sense to add tech preview for 8.17+ / 9.0? Same applies for the other functions

Reply:
can do it for 9.0 but we don't cover 8.x here